packages/protocol/contracts/shared/bridge/EtherBridgeWrapper.sol

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "./Bridge.sol";
+import "../../shared/based/ITaiko.sol";
+import "src/layer1/based/ITaikoInbox.sol";
+import "src/shared/libs/LibAddress.sol";
+import "src/shared/common/EssentialContract.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+contract EtherBridgeWrapper is EssentialContract {
```

Comment:
How about we call it EtherVault? If this contract can be merged into ERC20Vault and we consider sending Ether is just another special instance of sending ERC20 tokens with a special token address  `0x0`, then I'd strongly prefer that implementation.